lib/LaTeXML/Package.pm

```diff
@@ -2095,7 +2095,11 @@ sub loadTeXDefinitions {
     # since someone's presumably asking _explicitly_ for the raw TeX version.
     # It's probably even the ltxml version is asking for it!!
     # Of course, now it will be marked and wont get reloaded!
-    return if LookupValue($request . '_loaded');
+    #
+    # babel.sty exception:
+    # we know the same .ldf file may be reloaded with a different option,
+    # to load an adjacently defined language, so allow that.
+    return if $type ne 'ldf' and LookupValue($request . '_loaded');
```

Comment:
such specific special cases can cause lots of pain later on.  Perhaps would be best to keep track from the calls which can be loaded multiple times and which not?  Not sure of the logic yet, though.

Reply:
or at worst, a new `pathname_is_reloadable`, analogous to `pathname_is_raw`?

Reply:
Alright, I did the `at worst` suggestion, since it seemed easy and reasonable. I even moved `pathname_is_raw` and improved the docs a little, for extra cleanliness.